experiments.json

```diff
@@ -238,5 +238,20 @@
       "threshold": 0.2,
       "description": "Edit Top Sites is enabled"
     }
+  },
+  "pocketStories": {
+    "name": "Trending stories provided by Pocket",
+    "active": true,
+    "description": "Display recommendations of trending stories by Pocket",
+    "control": {
+      "value": true,
+      "description": "Pocket trending stories active"
+    },
+    "variant": {
+      "id": "exp-017-pocketstories",
+      "value": true,
+      "threshold": 0,
```

Comment:
@jaredkerim @k88hudson yes, my preference would be to merge this and send a separate PR next week to activate the experiment once we get everyone's OK. I am fine either way though.